minerl/Malmo/Minecraft/src/main/java/com/microsoft/Malmo/Client/MalmoModClient.java

```diff
@@ -208,43 +208,6 @@ public void onPressed()
         this.keyManager = new KeyManager(settings, extraKeys);
     }
 
-    /**
```

Comment:
How will we prevent locking out non-gui agents from moving?

Reply:
oh you make a good point, let me restore this bit of code. Deleted it accidentally with the debug function ^_^